tms/src/main/java/com/netcracker/edu/tms/dao/TaskDaoJpaImpl.java

```diff
@@ -0,0 +1,106 @@
+package com.netcracker.edu.tms.dao;
+
+import com.netcracker.edu.tms.model.Priority;
+import com.netcracker.edu.tms.model.Status;
+import com.netcracker.edu.tms.model.Task;
+
+import org.springframework.stereotype.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Query;
+
+import java.math.BigInteger;
+import java.util.Date;
+import java.util.List;
+
+@Repository
+public class TaskDaoJpaImpl implements TaskDao {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public Task getTaskById(BigInteger taskId) {
+        return entityManager.find(Task.class, taskId);
+    }
+
+    @Override
+    public List<Task> listOfTasksByName(String taskName) {
+        Query namesQuery = entityManager.
+                createQuery(String.format(QueryBuilder.selectQuery,"Task","taskName"));
```

Comment:
missed space between args
createQuery(String.format(QueryBuilder.selectQuery, "Task", "taskName"));

Reply:
fix this for all methods below